apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -652,8 +653,11 @@ if (
 	registerDelegateEventSubscriber(
 		'wpcom-block-editor-template-part-detach-blocks',
 		'before',
-		() => {
-			ignoreNextReplaceBlocksAction = true;
+		( mapping, event, target ) => {
+			const item = target.querySelector( '.components-menu-item__item' );
+			if ( item?.innerText === __( 'Detach blocks from template part' ) ) {
```

Comment:
@Automattic/i18n will this work, matching a translated string from Core? ^

Reply:
@creativecoder, Seems like it's using the global `@wordpress/i18n` instance in the same context as the block editor, so they are sharing the same locale data and the results of the `__` translate call in the menu item and in the condition should be matching.